Doc/c-api/call.rst

```diff
@@ -0,0 +1,344 @@
+.. highlight:: c
+
+.. _call:
+
+Call Protocol
+=============
+
+CPython supports two different calling protocols:
+*tp_call* and vectorcall.
+
+The *tp_call* Protocol
+----------------------
+
+Classes can implement callables by setting :c:member:`~PyTypeObject.tp_call`.
+
+A call is made using a tuple for the positional arguments
+and a dict for the keyword arguments, just like ``f(*args, **kwargs)``
+in the Python language.
+*args* must be non-NULL
```

Comment:
It doesn't matter in the built documentation. I personally prefer breaking such lines logically, instead of at an arbitrary number of characters. So I tend to use a newline at the end of a sentence. If this is considered bad style, I can fix it.